src/axon-support/src/main/java/engineering/everest/lhotse/axon/config/AxonConfig.java

```diff
@@ -104,7 +104,6 @@ public AnnotationCommandTargetResolver annotationCommandTargetResolver() {
 
     public enum EventProcessorType {
         SUBSCRIBING,
```

Comment:
Given the issues you identified for subscribing procoessors and the fact we don't expose a way to replay with it, should we also remove support for subscribing processor altogether? It can be a separate PR.

Reply:
It's available in Axon.... I don't recommend using it going forward.... maybe? :-)